se.emijoh.mpw.yml

```diff
@@ -0,0 +1,24 @@
+app-id: se.emijoh.mpw
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: mpwcli.py
+modules:
+  - name: mpwcli
+    buildsystem: simple
+    build-commands:
+      - install -D mpwcli.py /app/bin/mpwcli.py
+    sources:
+      - type: file
+        path: mpwcli.py
+  - python3-pyperclip.json
+  - python3-mpwalg.json
+  - name: wl-clipboard
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/bugaevc/wl-clipboard/archive/v2.0.0.tar.gz
+        sha256: 2c42f182432adabe56da0f1144d5fcc40b7aae3d8e14d2bc4dc4c3f91b51808d
+finish-args:
```

Comment:
Add `--socket=wayland`

Reply:
Thanks! 

I have tried adding the --socker=wayland but flatpak never added the permission as i did not add that extra `-` before each element in the yml list :facepalm: 

Now I can write to the cliboard!